chatbot/models.py

```
@@ -0,0 +1,506 @@
+        نمایش متنیِ قابل‌خواندن از جلسه چت‌بات شامل نوع جلسه، نام/نمایش کاربر و وضعیت.
+        
+        این رشته برای نمایش در رابط‌های مدیریتی، لاگ‌ها و هنگام تبدیل شی به رشته استفاده می‌شود و فرمت آن:
+        "جلسه {session_type} - {user} ({status})" است.
+        
+        Returns:
+            str: نمایش خلاصه و مختصر جلسه
+        """
+        return f"جلسه {self.session_type} - {self.user} ({self.status})"
+    
+    @property
+    def is_active(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `is_active`

(ANN201)